.devcontainer/Dockerfile

```
@@ -0,0 +1,9 @@
+# renovate: datasource=docker depName=mcr.microsoft.com/devcontainers/javascript-node
+FROM mcr.microsoft.com/devcontainers/javascript-node:22-bookworm@sha256:8ecd955dc231543929c46db04ab1a8962642bed804a84cdebb87a971dd5916d8
+
+# Install dependencies during build for faster dev container startup
+COPY package.json package-lock.json ./
```

Hadolint (2.14.0)

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)